mods/ra/maps/fort-lonestar/map.yaml

```diff
@@ -563,16 +577,31 @@ Rules:
 		CashTrickler:
 			Period: 250
 			Amount: 50
+	MOBILETENT:
+		Inherits: MCV
+		Transforms:
+			IntoActor: tent
+			Offset: 0,0
+		RenderSprites:
+			Image: MCV
+			Scale: 0.5
```

Comment:
`RenderSprites`->`Scale` is a bad idea and should never be used. It just promotes #3031 (and prolly other issues as well).